src/toggle.jsx

```diff
@@ -9,17 +9,70 @@ import ThemeManager from './styles/theme-manager';
 const Toggle = React.createClass({
 
   propTypes: {
+    /**
+     * The value of the toggle button. Is true when toggle
+     * has been turned on. False otherwise.
+     */
```

Comment:
Seems a bit confusing. Feels like a query-able property. 

*Determines whether the `Toggle` is initially turned on*